ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -287,6 +287,10 @@
                   - ../../build/build
 
     publishers:
+      - naginator:
+          max-failed-builds: 3
+          regular-expression: "Failure talking to yum|Call to virDomainLookupByUUID failed"
```

Comment:
For the libvirt error, we have a script for bringing vms up that you might want to try out:

https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/tests/functional/scripts/vagrant_up.sh

Reply:
thanks @alfredodeza